### DISCUSSION TITLE
Is it possible to launch Chrome on host (symlinked with distrobox-host-exec) with VSCode inside distrobox?

### DISCUSSION THREAD

**Discussion by regunakyle** [89luca89/distrobox#1376] [Q&A]
I ran `sudo ln -s /usr/bin/distrobox-host-exec /usr/local/bin/google-chrome` in a Debian 12 distrobox, then I tried to launch Chrome (with VSCode debug config) and failed. VSCode showed a popup `Unable  to launch browser: "Unable to attach to browser"`. 

VSCode can launch Chrome (or Chromium) installed inside the distrobox. Not sure what went wrong in the host symlink case.

Steps to reproduce:
1. Install VSCode in a Debian/Fedora distrobox ([Official guide](https://code.visualstudio.com/docs/setup/linux))
2. Install Chrome on the host
3. Inside the distrobox container, run `sudo ln -s /usr/bin/distrobox-host-exec /usr/local/bin/google-chrome`
4. Open a random folder with VSCode inside distrobox container, create a `.vscode` folder, then create a `launch.json` inside with the following content:
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Launch Chrome",
      "request": "launch",
      "type": "chrome",
      "url": "http://localhost:5173",
      "webRoot": "${workspaceFolder}"
    }
  ]
}
```
5. Press F5 to launch Chrome (and fails)
6. Install Chrome or Chromium inside the distrobox container, then press F5 again (You need to add `"runtimeExecutable": "/usr/bin/chromium"` under `webRoot` in `launch.json` if you use Chromium). Now the browser is launched properly.

**Comment by XandeFigueiredo** [comment]
I had some issues opening host browser links, but I found a workaround this way: https://github.com/89luca89/distrobox/issues/1984
